server/storage/database.go

```diff
@@ -124,16 +125,17 @@ func (db *SQLStorage) UpdateMany(gun string, updates []MetaUpdate) error {
 }
 
 // GetCurrent gets a specific TUF record
-func (db *SQLStorage) GetCurrent(gun, tufRole string) ([]byte, error) {
+func (db *SQLStorage) GetCurrent(gun, tufRole string) (*time.Time, []byte, error) {
 	var row TUFFile
-	q := db.Select("data").Where(&TUFFile{Gun: gun, Role: tufRole}).Order("version desc").Limit(1).First(&row)
+	q := db.Select("created_at, data").Where(
```

Comment:
Yep. And `created_at` is correct for GetChecksum due to the immutability aspect.